region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -1660,6 +1910,102 @@
         "h_CrystalFlash"
       ],
       "flashSuitChecked": true
+    },
+    {
+      "link": [5, 1],
+      "name": "Leave With Spark",
+      "startsWithShineCharge": true,
+      "requires": [
+        "canShinechargeMovementComplex",
+        {"shineChargeFrames": 120},
+        {"shinespark": {"frames": 9}}
```

Comment:
- There isn't a simple way to shinespark back out the door.
- Are shineChargeFrames lenienced at this step?  I don't want scaling to make a return shinespark impossible.